operator/roles/default/kiali-deploy/templates/kubernetes/deployment.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   labels:
     app: kiali
     version: {{ kiali_vars.deployment.version_label }}
+  annotations:
```

Comment:
are you sure you want this here on the deployment itself? Or do you want it on the actual pod? (see template section below)

Reply:
it's the deployment in this case